.gitignore

```diff
@@ -68,3 +68,6 @@ fastlane/readme.md
 
 #extra
 *.DS_Store
+
+#VS Code
+.project

```

Comment:
you need to revert this file first, than it will ignore it for commits

Reply:
reverted and squashed.